### ISSUE TITLE
Optionally expose all underlying container metadata in status

### ISSUE THREAD

**Issue by cgwalters** [containers/bootc#348]
As someone deploying a bootc system, I may want to add custom labels (or OCI annotations) into my image, and I want to be able to see that on the client side to potentially act on it.

Today this is all squirreled away in the underlying ostree-container store, one can extract it with `ostree container metadata`, but that's awkward and we want to get away from exposing ostree (this also relates to https://github.com/containers/bootc/pull/215 )

Maybe what we do for now is something like `bootc status --full` which would dump the container manifest and config into the image status?

**Comment by cgwalters** [comment]
Or maybe it would be better as `bootc image inspect` or so?  If we put it in status, it could get really unwieldy if we have both current and rollback, plus any future [lifecycle bound images](https://github.com/containers/bootc/issues/128).

Maybe we should have `bootc image list` and `bootc image inspect` in that vein etc.

**Comment by cgwalters** [comment]
> would bootc edit support adding custom metadata?

Hmm what would be the use case for that?  Adding container metadata is a build-time thing right?

> @cgwalters what if we namespace the labels with something like ostree.status.whatever... when building the images so we avoid cluttering rpm-ostree status with too many stuff and only show those labels?

The question of exposing container information in `rpm-ostree status` is something we should probably track in that git repo, not here.  It has a similar design shape issue around having it out of band vs inline and size.

**Comment by runcom** [comment]
> The question of exposing container information in `rpm-ostree status` is something we should probably track in that git repo, not here. It has a similar design shape issue around having it out of band vs inline and size.

ack, I wrongly thought this was for rpm-ostree status, but it's bootc status, nevermind :)

**Comment by sallyom** [comment]
> > would bootc edit support adding custom metadata?
> 
> Hmm what would be the use case for that? Adding container metadata is a build-time thing right?
> 
right, I was imagining that `bootc edit` would perform something like a `podman commit` but didn't really think it all the way through - that would get confusing
